probe/endpoint/procspy/background_reader_linux.go

```diff
@@ -0,0 +1,133 @@
+package procspy
```

Comment:
Darwin doesn't have namespaces nor does it read `/proc`, which is the reason for doing the rate-limiting.